functions/Restore-DbaDatabase.ps1

```diff
@@ -463,7 +518,7 @@ function Restore-DbaDatabase {
 		#$BackupFiles 
 		#return
 		Write-Message -Level Verbose -Message "sorting uniquely"
-		$AllFilteredFiles = $backupFiles | sort-object -property fullname -unique | Get-FilteredRestoreFile -SqlInstance $SqlInstance -RestoreTime $RestoreTime -SqlCredential $SqlCredential -IgnoreLogBackup:$IgnoreLogBackup -TrustDbBackupHistory:$TrustDbBackupHistory -Silent:$Silent
+		$AllFilteredFiles = $backupFiles | sort-object -property fullname -unique | Get-FilteredRestoreFile -SqlInstance $SqlInstance -RestoreTime $RestoreTime -SqlCredential $SqlCredential -IgnoreLogBackup:$IgnoreLogBackup -TrustDbBackupHistory:$TrustDbBackupHistory -Silent:$Silent -continue:$continue -ContinuePoints:$ContinuePoints -DatabaseName $DatabaseName
```

Comment:
Sort-Object -Property -Unique with caps is easier to read but I hate to burden you with more things. Just for the future.